src/System.Private.CoreLib/shared/System/Globalization/SortKey.cs

```diff
@@ -2,37 +2,25 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-////////////////////////////////////////////////////////////////////////////
-//
-//
-//  Purpose:  This class implements a set of methods for retrieving
-//            sort key information.
-//
-//
-////////////////////////////////////////////////////////////////////////////
-
-using System;
-using System.Runtime.CompilerServices;
 using System.Diagnostics;
 
 namespace System.Globalization
 {
+    /// <summary>
+    /// This class implements a set of methods for retrieving
+    /// </summary>
     public partial class SortKey
     {
-        //--------------------------------------------------------------------//
-        //                        Internal Information                        //
-        //--------------------------------------------------------------------//
+        private readonly string _localeName;
 
-        internal string _localeName;       // locale identifier
+        internal readonly CompareOptions _options;
```

Comment:
Nit: Is it intentionally left as `internal` and not `private`? I am not able to find the place where it is used outside of this file.

Reply:
Yes it is used I found

Reply:
Where is it used? When I change it to private and try to build CoreLib, both Windows and Unix builds work fine.

Reply:
I believe this not used outside the class. I tried to search through the source index and couldn't find any hit. I agree with Jan, just change it to private and try to build Windows/Linux.